### DISCUSSION TITLE
Welcome, Alienegra!

### DISCUSSION THREAD

**Discussion by AlienegraGeek** [ParanoidUser/yolo#529] [Subscribe]
Walk it like I talk it...
Captain Pirate, I believe we are teammates...
So I can get the collaborator permission's too!

**Comment by ParanoidUser** [comment]
Yo ho heave ho, @AlienegraGeek! Welcome aboard!

The invitation to join the collaborator group has been sent to you. Thanks for your interest in this project!
Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers
